docs/en/ingest-management/elastic-agent/configuration/generate-standalone-agent-config.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[generate-standalone-agent-config]]
+= Generate a standalone {agent} configuration file
```

Comment:
Another title idea is "Get started with standalone {agent}"? I noticed we don't have anything like a quickstart for standalone agent yet, but this could potentially serve as one?

Reply:
We are trying to avoid the proliferation of "getting started" topics so that someday we'll have a clear getting started path for the stack, so I think I will not change the title right now.